packages/trader/src/Modules/Trading/Containers/allow-equals.jsx

```diff
@@ -29,6 +29,7 @@ const AllowEquals = ({ onChange, is_allow_equal, has_equals_only, className }) =
 };
 
 AllowEquals.propTypes = {
+    className: PropTypes.string,
```

Comment:
As this is prop here, so I think it is better to be `class_name`.

Reply:
I guess this one is ok